### ISSUE TITLE
跑cnn_valid.py出错

### ISSUE THREAD

**Issue by zhaolonglongywx** [zhangjiali1201/Keras_bearing_fault_diagnosis#1]
您好，我在跑cnn_valid.py的时候弹出了FileNotFoundError: [Errno 2] No such file or directory: 'images/1-conv-32-filters-0-dense.png'
对应到代码中的训练模型那块的这段代码
            model.fit(x=x_train, y=y_train, batch_size=batch_size, epochs=epochs,
                      verbose=1, validation_data=(x_valid, y_valid), shuffle=True,
                      callbacks=[tb_cb])

            plot_model(model=model,
                       to_file="images/{}-conv-{}-filters-{}-dense.png".format(conv_layer, layer_size, dense_layer),
                       show_shapes=True)
求解答

**Comment by zhangjiali1201** [comment]
报错是因为保存图片的时候找不到这个文件夹，看下你是不是把images文件夹删掉了，再建一个空的
